### PR TITLE
Add more translations

### PR DESCRIPTION
**Pull request by fbonzon**
[Live demo](http://françois.ch/graphhopper/?locale=fr-CH&layer=OpenStreetMap)

### CODE REVIEW

File: web/src/main/webapp/js/map.js [hunk 1]
```diff
@@ -164,12 +164,12 @@ function initMap(bounds, setStartCoord, setIntermediateCoord, setEndCoord, selec
         },
         contextmenu: true,
         contextmenuItems: [{
-                text: 'Route ',
```
Comment (karussell) [inline web/src/main/webapp/js/map.js]: here is a space after 'Route' ... maybe we should use a parametrized translation. And did you add those keys to they spreadsheet? If not, I can do it.

**Comment by karussell** [comment]
Thanks - looks good!

File: web/src/main/webapp/js/map.js (hunk 1 above)
Comment (fbonzon) [inline web/src/main/webapp/js/map.js]: Added the space after "Route".

Same question about parametrized translation can be raised below for keys "start_label", "intermediate_label", "end_label", where a number is added after the translated string + space.

I don't think parametrized translation is necessary here.

Yes, all these keys were already added on the spreadsheet a few months ago.

**Comment by fbonzon** [comment]
Addressed your comment.

**Comment by karussell** [comment]
Thanks!

**Pull request merged**